src/edge/tests/test_genome.py

```diff
@@ -383,3 +383,101 @@ def test_get_changed_locations_by_fragment(self):
                 )
             else:
                 raise Exception("Unexpected fragment")
+
+    def test_get_coordinate_diff_from_parent_genome(self):
```

Comment:
can you add a test case of adding something to the start of the fragment, and also a test case where you removed the last few bases from a fragment?

Reply:
I'm actually not sure if this handles those cases for now, so I'll also add that handling in and write up a test